src/main/java/org/linksharing/server/http/ApiRestController.java

```diff
@@ -66,15 +68,15 @@ ResponseEntity<UserProfileDetails> updateProfileDetails(Principal user, @Request
     }
 
     @GetMapping("/profile/picture")
-    ResponseEntity<?> getProfilePicture(Principal user) throws FileNotFoundException {
+    ResponseEntity<?> getProfilePicture(Principal user) throws IOException {
 
         UserProfileDetails userProfile = profileRepository.findByEmail(user.getName());
         String imageUrl;
 
         if (userProfile.getImageUrl() != null) {
             imageUrl = userProfile.getImageUrl();
         } else  {
-            imageUrl = "../image.jpg";
+            imageUrl = "src/main/resources/static/placeholder.jpg";
```

Comment:
Have you tried to run the app as standalone jar file and check if this works?

Reply:
Nope. But if you ask then I suppose it won't.
I didn't know what path is correct and calculated that one like this:
`new File("").toURI().relativize(new File("D:/Java_Projects/link-sharing-app/src/main/resources/static/placeholder.jpg").toURI()).getPath();`
It should be relative but looks like absolut which is a little odd. 
Anywat it works at the moment, so I put it on hold for now.

Reply:
When running jar file on server you will get this:
> java.io.FileNotFoundException: src\main\resources\static\placeholder.jpg

Use `org.springframework.core.io.ClassPathResource` with correct path within class path.